Upstream: No
Reason: We build py3 without gettext support, so these functions dont
exist
diff --git a/GTG/core/translations.py b/GTG/core/translations.py
index 6ff63a7..866e1be 100644
--- a/GTG/core/translations.py
+++ b/GTG/core/translations.py
@@ -30,9 +30,7 @@ GETTEXT_DOMAIN = 'gtg'
 LOCALE_PATH = gettext.bindtextdomain(GETTEXT_DOMAIN)
 
 gettext.bindtextdomain(GETTEXT_DOMAIN, LOCALE_PATH)
-locale.bindtextdomain(GETTEXT_DOMAIN, LOCALE_PATH)
 gettext.textdomain(GETTEXT_DOMAIN)
-locale.textdomain(GETTEXT_DOMAIN)
 
 translation = gettext.translation(GETTEXT_DOMAIN, LOCALE_PATH, fallback=True)
 
